main.js

```diff
@@ -200,6 +206,23 @@ export class IdEncoder {
    * @returns {string} - Encoded string.
    */
   encode(bytes) {
+    if(this.multihash) {
+      const byteSize = bytes.length;
+
+      if(byteSize > 127) {
+        throw new Error(`Identifier size too large.`);
```

Comment:
Fix quotes and use RangeError.
```suggestion
        throw new RangeError('Identifier size too large.');
```

Reply:
Updated fcd0ad9